### PR TITLE
OCPBUGS-34387: maintain session across tabs

### PR DESCRIPTION
**Pull request by rhamilto**

### CODE REVIEW

File: frontend/public/components/masthead-toolbar.jsx [hunk 1]
```diff
@@ -53,6 +53,8 @@ import { FeedbackModal } from '@patternfly/react-user-feedback';
 import { useFeedbackLocal } from './feedback-local';
 import feedbackImage from '@patternfly/react-user-feedback/dist/esm/images/rh_feedback.svg';
 
+const LAST_CONSOLE_ACTIVITY_TIMESTAMP_LOCAL_STORAGE_KEY = 'last-console-activity-timestamp';
```
Comment (jhadvig) [inline frontend/public/components/masthead-toolbar.jsx:56]: just a though to align with the actual feature:
```suggestion
const LAST_INACTIVITY_TIMEOUT_RESET_TIMESTAMP_LOCAL_STORAGE_KEY = 'last-inactivity-timeout-reset-timestamp';
```

**Review by jhadvig — APPROVED** [review]
Once comment, but otherwise the change looks good 🙌 
/lgtm

File: frontend/public/components/masthead-toolbar.jsx (hunk 1 above)
Comment (rhamilto) [reply frontend/public/components/masthead-toolbar.jsx:56]: 👍 Naming is hard.

**Pull request merged**